lib/user.ts

```diff
@@ -6,9 +10,14 @@ const displayNameElement = document
 	.getElementsByTagName('head')[0]
 const displayName = displayNameElement ? displayNameElement.getAttribute('data-user-displayname') : null
 
+const isAdmin = (typeof OC === 'undefined')
+	? false
+	: OC.isUserAdmin()
```

Comment:
```
lib/user.ts(15,7): error TS2339: Property 'isUserAdmin' does not exist on type 'Nextcloud.v16.OC | Nextcloud.v17.OC | Nextcloud.v18.OC | Nextcloud.v19.OC'.
```

Reply:
My fault. I did not understand why you added it to the common and v17 typings. But this is actually a bit of duplicated code due to minor differences … https://github.com/nextcloud/nextcloud-typings/pull/34#discussion_r433953512 was a mistake. Could you send a PR to fix that?

Sorry

Reply:
Sure, will do. No worries

Reply:
v0.2.2 of the typings should work :pray:

Reply:
Updated